src/main/java/spoon/support/compiler/jdt/JDTBatchCompiler.java

```diff
@@ -94,21 +94,21 @@ public JDTBatchCompiler(JDTBasedSpoonCompiler jdtCompiler) {
 						int lastSlash = CharOperation.lastIndexOf(File.separatorChar, charName);
 						if (lastSlash != -1) {
 							char[] modulePath = CharOperation.subarray(charName, 0, lastSlash);
-							pathToModCU.put(String.valueOf(modulePath), compilationUnit);
 
 							lastSlash = CharOperation.lastIndexOf(File.separatorChar, modulePath);
 							if (lastSlash == -1) {
 								lastSlash = 0;
 							} else {
 								lastSlash += 1;
 							}
-
+							//TODO the module name parsed by JDK compiler is in `this.modNames`
```

Comment:
Why keeping this as a todo? If I understand well it simplifies the work for getting the name, no?

Reply:
but we need to found correct record. It is possible, but it needs few minutes more development 
effort ;-)

Reply:
Oh I thought you already identified it properly. So ok leave it like that for now.